lib/netext/httpext/error_codes.go

```diff
@@ -35,6 +35,10 @@ import (
 	"golang.org/x/net/http2"
 )
 
+// TODO: mave rename the type errorCode, so we can have errCode variables? and
```

Comment:
I'm confused? In go, we have the default `error` interface and you usually name any variables `err`. So this is sort of the reverse - the type name is the shortened `errCode`, so the more common variables would have to be named the longer and unidiomatic `errorCode` ...

Reply:
https://godoc.org/golang.org/x/net/http2#ErrCode

Reply:
¯\_(ツ)_/¯ consistency FTW... :man_facepalming: Having written some error handling code, I'd still prefer if we rename these type and constants on the k6 side... Both the prefix (see how even the http2 constants are prefixed with `ErrCode`, not suffixed) and the `errCode` to `errorCode` (or `ErrorCode`, when we make error codes not specific to HTTP)

Reply:
(both of these can wait, that's why they're as a TODO, maybe we can do it when we fix the too common `1000` error code we noticed)

Reply:
The problem with the *too common* `1000` is that the stdlib is returning just plain stringErrors instead of the actual errors that are defined in the http2 package (for example). Which means that we will need string matching ... but yeah I propose we all of those changes when we again touch those instead of in this PR

Reply:
Added a separate issue: https://github.com/loadimpact/k6/issues/1104